sdk/textanalytics/Azure.AI.TextAnalytics/tests/DetectLanguageTests.cs

```diff
@@ -249,5 +232,50 @@ public async Task DetectLanguageBatchWithNullTextTest()
             InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => results[0].PrimaryLanguage.GetType());
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
+
+        private void ValidateInDocumenResult(DetectedLanguage language)
+        {
+            Assert.That(language.Name, Is.Not.Null.Or.Not.Empty);
+            Assert.That(language.Iso6391Name, Is.Not.Null.Or.Not.Empty);
+            Assert.GreaterOrEqual(language.ConfidenceScore, 0.0);
+        }
+
+        private void ValidateBatchDocumentsResult(DetectLanguageResultCollection results, bool includeStatistics = default)
+        {
+            Assert.IsNotEmpty(results.ModelVersion);
+
+            if (includeStatistics)
+            {
+                Assert.IsNotNull(results.Statistics);
+                Assert.Greater(results.Statistics.DocumentCount, 0);
+                Assert.Greater(results.Statistics.TransactionCount, 0);
+                Assert.GreaterOrEqual(results.Statistics.InvalidDocumentCount, 0);
+                Assert.GreaterOrEqual(results.Statistics.ValidDocumentCount, 0);
+            }
+            else
+                Assert.IsNull(results.Statistics);
+
+            foreach (DetectLanguageResult languageInDocument in results)
```

Comment:
Is it necessary to assert that at least one result has been returned?

Reply:
yes. good point. will add